utils/tx.ts

```diff
@@ -464,15 +466,26 @@ export const sellAsset = async (
   minAmountOut: BN,
   options = {},
 ) => {
-  const mangata = await getMangataInstance();
-  return await mangata.xyk.multiswapSellAsset({
-    account: account,
-    tokenIds: [soldAssetId.toString(), boughtAssetId.toString()],
-    amount: amount,
-    minAmountOut: minAmountOut,
-    txOptions: options,
-  });
+  let liqId: BN;
+  const api = getApi();
+  liqId = await getLiquidityAssetId(soldAssetId, boughtAssetId);
+  if (liqId < BN_ZERO) {
+    liqId = BN_MILLION;
+  }
```

Comment:
what is this?  
& second, BNs can not be compared with integer operators ( < ) , use the bnLT funcs!

Reply:
This is a way how we can receive error "NoSuchPool". Fixed